vm/src/obj/objobject.rs

```diff
@@ -237,7 +237,7 @@ pub fn get_attributes(obj: &PyObjectRef) -> PyAttributes {
 
     // Get instance attributes:
     if let Some(dict) = &obj.dict {
-        for (key, value) in dict.get_key_value_pairs() {
+        for (key, value) in dict.into_iter() {
```

Comment:
I think the `into_iter()` is redundant, `for (key, value) in dict` should be desugared to the same thing